src/app/collection/settings/tab/attributes/collection-attributes.component.html

```diff
@@ -1,3 +1,86 @@
-<div class="mx-3">
-  <attribute-list [collection]="collection"></attribute-list>
+<div class="my-4">
+  <table class="table table-striped">
+    <thead [style.background-color]="lightenColor(collection.color)">
+      <tr class="d-flex">
+        <th class="col-1"></th>
+        <th class="col-5 d-flex align-items-center">
+          <span i18n="@@collections.tab.attributes.attribute">Attribute</span>
+        </th>
+        <th class="col-2 d-flex align-items-center">
+          <span i18n="@@collections.tab.attributes.usage">Usage</span>
+        </th>
+        <th class="col-4">
+          <div class="form-control d-flex align-items-center overflow-hidden" title="Search for attribute" i18n-title="@@collections.tab.attributes.search.title">
+            <input type="search" class="flex-grow-2" #searchInput
+                   width="100"
+                   placeholder="Search attribute..."
+                   i18n-placeholder="@@collections.tab.attributes.search.placeholder"
+                   [(ngModel)]="searchString">
+          </div>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="d-flex">
+        <td class="col-1"></td>
+        <td class="col-8">
+          <input-box
+            [canStayEmpty]="true"
+            [placeholder]="attributePlaceholder"
+            [initialValue]="newAttributeName"
+            (newValue)="newAttributeName = $event">
+          </input-box>
+        </td>
+        <td class="col-3">
+          <button type="button"
+                  class="btn btn-sm btn-success float-right"
+                  (click)="onCreateAttribute()">
+            <i class="fas fa-plus"></i>
+            <span class="font-weight-bold" i18n="@@collection.tab.attributes.attribute.add">Add attribute</span>
+          </button>
+        </td>
+      </tr>
+      <tr class="d-flex" *ngFor="let attribute of attributes | attributeFilter:searchString">
+        <th scope="row"
+            class="icon-hover col-1 d-flex align-items-center justify-content-center"
+            [class.visible]="isDefaultAttribute(attribute)">
+          <i *ngIf="isDefaultAttribute(attribute); else notDefaultIcon"
```

Comment:
Since this table might be quite big and often redrawn by search, I would recommend to create a pipe for getting the default attribute and use it here. `isDefaultAttribute()` method is called several times in this template.

Reply:
Good idea.

Reply:
Can you bring me some example how to use pipe in condition?

Reply:
I think the `else` block here is just too complicated. You can have `<i *ngIf="attribute | isDefaultAttribute:collection">` for the first one and `<i *ngIf="!(attribute | isDefaultAttribute:collection)">` for the second one.

Reply:
But maybe your solution has an advantage of executing it just once.